recipe/vs2010_omp.patch

```diff
@@ -0,0 +1,12 @@
+--- Source/LibRawLite/libraw/libraw_types.h
++++ Source/LibRawLite/libraw/libraw_types.h
+@@ -32,7 +32,7 @@
+ #if defined (_OPENMP)
+ 
+ #if defined(WIN32)
+-# if defined (_MSC_VER) && (_MSC_VER >= 1600 || (_MSC_VER == 1500 && _MSC_FULL_VER >= 150030729) )
++# if defined (_MSC_VER) && (_MSC_VER > 1600 || (_MSC_VER == 1500 && _MSC_FULL_VER >= 150030729) )
```

Comment:
Oops, must have not seen it when it went in. Though I guess I wasn't paying to close of attention to the Windows stuff as I was trusting you guys figured out what worked. This may very well be necessary just figured we should understand why and if it is not remove it.